config/application.rb

```diff
@@ -124,9 +124,8 @@ class Application < Rails::Application
       #   dev/test/prod in the config revamp.
       ActiveRecord::Base.logger.level = Logger::DEBUG
 
-      Rails::ConsoleMethods.class_eval do
-        include Vmdb::ConsoleMethods
-      end
+      Rails::ConsoleMethods.include(Vmdb::ConsoleMethods)
+      TOPLEVEL_BINDING.eval('self').extend(Vmdb::ConsoleMethods) if defined?(Pry)
```

Comment:
You could as a hook in your own .pryrc

TBF since this is a very specific tool that I probably use more than anyone else, I wouldn't be offended if you'd rather leave it out here (so people would have to know to add the hook themselves). Maybe I'll just fix it upstream later.